assets/css/icons.css

```diff
@@ -1,82 +1,117 @@
 @font-face {
-	font-family: 'thronesdb';
-	src:url('../fonts/thronesdb.eot?irjt2b');
-	src:url('../fonts/thronesdb.eot?#iefixirjt2b') format('embedded-opentype'),
-		url('../fonts/thronesdb.ttf?irjt2b') format('truetype'),
-		url('../fonts/thronesdb.woff?irjt2b') format('woff'),
-		url('../fonts/thronesdb.svg?irjt2b#thronesdb') format('svg');
-	font-weight: normal;
-	font-style: normal;
+  font-family: 'thronesdb';
+  src:  url('fonts/thronesdb.eot?pg0qpl');
```

Comment:
this change breaks the path to the font file (same with the `src:url(...)` entries below. please revert the path changes, thanks.